src/action/services/useFuncOfStorage.js

```diff
@@ -0,0 +1,44 @@
+import useLocalStorage from "./useLocalStorage.js";
+
+function useFuncOfStorage() {
+  const localStorageHandler = useLocalStorage();
+
+  const getMovieDetailsById = (id) => {
```

Comment:
Let's think togather about alternative writing style. this one is not readable and contains fair amoung of useless utils.

Reply:
no need to think togather, i modified uselocalstorage to have 2 exported functions
and did import * as useLocalStorage from... in every place in my code so hopefully 
the code now is less  not readable and contains fair amoung of useless